src/Nethermind/Nethermind.Ssz/Ssz.BitArray.cs

```diff
@@ -1,70 +0,0 @@
-// SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
```

Comment:
Basic ssz methods, let it stay

Reply:
I mean, it may be not useful for now, but looks like withdrawals and 4844 force us to have a small SSZ library anyway. Let these potentially useful primitives to stay for a bit more time.